blocks/common/button/button.css

```diff
@@ -0,0 +1,25 @@
+.button
+{

```

Comment:
Oh, is that style of putting `{` on the next line after selector from some config or just personal preference?